composer/models/huggingface.py

```diff
@@ -31,6 +36,8 @@ class HuggingFaceModel(ComposerModel):
     Args:
         model (transformers.PreTrainedModel): A 🤗 Transformers model.
         tokenizer (transformers.PreTrainedTokenizer, optional): The tokenizer used to prepare the dataset. Default ``None``.
+                                                                Note: If the tokenizer is provided, its config will be saved in the composer checkpoint, and it can be reloaded
```

Comment:
should be note syntax? `.. note::`, see Trainer for reference

Reply:
ah thanks will do